### PR TITLE
[feature]: add CSD-RTGS stub for link to Matthias

### PR DESCRIPTION
**Pull request by appetrosyan**
Added a stub article with almost empty contents to provide link to Matthias.

### CODE REVIEW

File: src/guide/reports/csd-rtgs.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# CSD/RTGS linkages Proof of concept 
+
+In this document we will describe the CSD/RTGS linkages PoC execution via Iroha. This is a project which was done in collaboration with the Asian Development Bank, and Fujitsu. Other participants used technologies such as R3 Corda, Hyperledger Cactus (Cacti), Hyperledger Fabric and many other popular blockchain solutions.
```
Comment (WRRicht3r) [inline src/guide/reports/csd-rtgs.md:3]:
```suggestion
In this document, we will describe the CSD/RTGS linkages PoC execution via Iroha. This is a project which was done in collaboration with the Asian Development Bank and Fujitsu. Other participants used technologies such as R3 Corda, Hyperledger Cactus (Cacti), Hyperledger Fabric, and many other popular blockchain solutions. 
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/guide/reports/csd-rtgs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (6r1d) [reply src/guide/reports/csd-rtgs.md:3]: This document will describe the CSD/RTGS linkages PoC execution via Iroha. CSD/RTGS is a project in collaboration with the Asian Development Bank and Fujitsu. Other participants used R3 Corda, Hyperledger Cactus (Cacti), Hyperledger Fabric and many other popular blockchain solutions.

**Review by 6r1d — CHANGES_REQUESTED** [review]
I want to change the text a bit.

File: src/guide/reports/csd-rtgs.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (appetrosyan) [reply src/guide/reports/csd-rtgs.md]: I'll replace this wholesale.

**Pull request merged**